framework/src/controller/channels/in_memory_channel.ts

```diff
@@ -21,10 +21,10 @@ import * as JSONRPC from '../jsonrpc/types';
 export class InMemoryChannel extends BaseChannel {
 	private bus!: Bus;
 
-	public async registerToBus(bus: Bus): Promise<void> {
+	public registerToBus(bus: Bus): void {
 		this.bus = bus;
 
-		await this.bus.registerChannel(this.moduleAlias, this.eventsList, this.actions, {
+		this.bus.registerChannel(this.moduleAlias, this.eventsList, this.actions, {
```

Comment:
Would be nice if we add condition to not register a channel only once.

Reply:
I added a check in registerChannel to throw error on a duplicate registerChannel call